examples/plot_ols_lasso_enet.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from scipy.linalg import block_diag
+import matplotlib.pyplot as plt
+
+from celer import ElasticNetCV, LassoCV
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import KFold
+
+
+n_samples, n_features, n_groups, corr, snr = 10000, 200, 10, 0.8, 2
+n_features_group = n_features // n_groups
+
+# build corr matrix
```

Comment:
X ~ multinomial with a block diag cov matrix
a block represents a group of correlated features